tests/FilterTest.php

```diff
@@ -23,6 +23,25 @@ public function testParserIdentifiesSingleConditions()
         );
     }
 
+    public function testParserIdentifiesWildCardCharacters()
```

Comment:
This test is identical to the new cases in `testParserIdentifiesRelationalOperators`. Why not change it to check if the parser correctly differentiates between the condition types if there's a wildcard and if there's none?

Reply:
Changed the test as per your suggestion.